collections/eurocrops.yaml

```diff
@@ -0,0 +1,135 @@
+GeographicalCoverage:
+  - Austria
+  - Belgium
+  - Germany
+  - Denmark
+  - Estonia
+  - Spain
+  - France
+  - Croatia
+  - Lithuania
+  - Latvia
+  - Netherlands
+  - Portugal
+  - Romania
+  - Sweden
+  - Slovenia
+  - Slovakia
+TemporalAvailability:
+  Table:
+    Columns:
+      - Name: Name
+        Title: Country
+      - Name: Description
+        Title: Year
+    Rows:
+      - Name: Austria
+        Description: 2021
+      - Name: Belgium
+        Description: 2021
+      - Name: Germany
+        Description: 2021
+      - Name: Denmark
+        Description: 2019
+      - Name: Estonia
+        Description: 2021
+      - Name: Spain
+        Description: 2020
+      - Name: France
+        Description: 2018
+      - Name: Croatia
+        Description: 2020
+      - Name: Lithuania
+        Description: 2021
+      - Name: Latvia
+        Description: 2021
+      - Name: Netherlands
+        Description: 2021
+      - Name: Portugal
+        Description: 2021
+      - Name: Romania
+        Description: Data is collected across different years.
+      - Name: Sweden
+        Description: 2021
+      - Name: Slovenia
+        Description: 2021
+      - Name: Slovakia
+        Description: 2021
+  
+UpdateFrequency: On-demand
+Attributes:
+  Table:
+    Columns:
+      - Name: Name
+        Title: Name
+      - Name: Description
+        Title: Description
+      - Name: Type
+        Title: Type
+    Rows:
+      - Name: aoi
+        Description: Area of interest
+        Type: Polyon or bounding box
+      - Name: time interval
+        Description: A time interval to retrieve Sentinel-2 data
+        Type: String
+      - Name: evalscript
+        Description: An evalscript specifying which Sentinel-2 bands or derived indices, e.g. NDVI, to download
+        Type: String
+      - Name: Bucket name
+        Description: Required AWS S3 Bucket name, must be in eu-central-1
+        Type: String
+      - Name: Project folder
+        Description: The location of the project folder on the provided AWS S3 bucket
+        Type: String
+      - Name: AWS access key ID
+        Description: The AWS access key ID of the IAM user with read and write access to the bucket
+        Type: String
+      - Name: AWS secret access key
+        Description: The AWS secret access key of the IAM user with read and write access to the bucket
+        Type: String
+ProducedResults: |
+  The produced output are [Sentinel Hub Statistical API](https://docs.sentinel-hub.com/api/latest/api/statistical/) results for each parcel saved to the user's S3 bucket in the form of JSON files.
+Pricing: |
+  Calculation the price of an order is proportionate to the amount of data that will have to be processed. Processing Units (PUs) for this algorithm are directly proportional to:
```

Comment:
Small typo 

Calculation *of* the price....

Reply:
Fixed.